services/myget/myget.tester.js

```diff
@@ -32,21 +32,21 @@ t.create('total downloads (valid)')
 
 t.create('total downloads (not found)')
   .get('/mongodb/dt/not-a-real-package.json')
-  .expectJSON({ name: 'downloads', value: 'not found' })
+  .expectJSON({ name: 'downloads', value: 'package not found' })
 
```

Comment:
The next two tests are failing because of the way we're propagating errors through `regularUpdate`. It might just be a problem with `nock`. I have to think it through a bit more.

Reply:
I don't think this is an issue with `nock`. If you set `apiBaseUrl` to something that won't resolve and test it through the front end, we throw an `UnhandledPromiseRejectionWarning` rather than rendering an "inaccessable" badge like we do with other badges.